contrib/kvm-setup/roles/kvm-setup/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: Update apt
+  raw: apt-get update
+  when: bootstrap_os in ['Ubuntu', 'Debian']
```

Comment:
We don't need to do it

- name: Basic packages required by ansible
  apt: >
    upgrade: yes
    update_cache: yes
    cache_valid_time: 3600
    name: {{ item }}
    with_items:
       - python
       - python-apt